### PR TITLE
Simpler is better ?

### PR DESCRIPTION
**Pull request by gluck**
Inspired by storybook vue3 renderer:
https://github.com/storybookjs/storybook/blob/next/app/vue3/src/client/preview/render.ts

It's so simple it must be broken somehow but I didn't manage to break it ....

### CODE REVIEW

File: src/typeOf.ts [hunk 1]
```diff
@@ -38,7 +35,7 @@ export function typeOf(obj: any): string {
     if ("Component" in obj && typeOf(obj.Component) === "Svelte") {
       return "SvelteStory";
     }
-    if ("components" in obj && ("template" in obj || "render" in obj)) {
+    if ("__v_isVNode" in obj || "__scopeId" in obj || ("components" in obj && ("template" in obj || "render" in obj))) {
```
Comment (gluck) [inline src/typeOf.ts:38]: scopeId allows support of simpler vue story:
```
export const story = () => MyCounter;
```

**Review by gqio — APPROVED** [review]

File: src/typeOf.ts (hunk 1 above)
Comment (georges-gomes) [reply src/typeOf.ts:38]: Works from you have 2 stories and you switch from one to the other ?

Comment (gluck) [reply src/typeOf.ts:38]: Yes

Comment (georges-gomes) [reply src/typeOf.ts:38]: Let's go then

**Review by muryoh — APPROVED** [review]

**Pull request merged**